### DISCUSSION TITLE
Device not booting after web flash

### DISCUSSION THREAD

**Discussion by mroggi** [toverainc/willow#293] [Q&A] [answered]
Dear community, I have successfully flashed my S3-Box via the web flasher. After uploading 100% and verification it said: upload successful, restarting device. 
After that the display flashes and then stays black. The device does not show up in WAS. Also tried to flash other versions but same result.
Any ideas what I can do to reCtivate the device?
Thanks!!!

**Comment by hamishcunningham** [comment]
is it the same problem as https://github.com/toverainc/willow/discussions/280#discussioncomment-7165974 maybe?

edit: oops, probably not, I thought you were flashing a box*3*

**Comment by nikito** [comment]
If you touch the screen does the device wake up?

**Reply by mroggi** [reply]
No, no reaction. It does not look like its booting. But I can reflash it

**Comment by kristiankielhofner** [comment, marked as answer]
@mroggi - Can you try our [preview web flasher](https://flash-preview.heywillow.io/)? It shows more serial output. Just make sure you select "ESP32-S3-Box" as this uses the BOX-3 by default. When you flash with it and the device boots, can you copy and paste the full log output (as text) here?

**Reply by mroggi** [reply]
Thank you, that worked! Also used a different machine, not sure what made the difference.
Thanks for the quick help!!!

**Reply by kristiankielhofner** [reply]
That's great to hear! flash-preview runs from the main branch of our web flasher repo so it is a preview of the next stable release of the web flasher. It will be the default fairly soon although I suspect the real difference here was your alternate machine because they are fundamentally the same thing.
